### ISSUE TITLE
role alert not working: JAWS + IE combination

### ISSUE THREAD

**Issue by ktekku** [accessibilitysupported/a11ysupport.io#87]
Screenreader does not read alert message. Why, how to solve this?
JAWS + Chrome works well.

**Comment by mfairchild365** [comment]
Hi @ktekku - support for the alert role is not fully documented yet. What aspect of support is not working in JAWS + IE?

**Comment by ktekku** [comment]
It seems to be that the problem is caused by our system.
